### PR TITLE
refactor: added text-utilities class

### PR DESCRIPTION
**Pull request by tanishq20**
- Added typography content
- Added text-utilities class
- Defined global variables for text

Live preview link: https://deploy-preview-26--compact-ui.netlify.app/docs/customization/typography
preview image

### CODE REVIEW

File: stylesheets/abstracts/variables.css [hunk 1]
```diff
@@ -43,4 +43,33 @@
   --github-color: black;
```
Comment (shubhamkumar648) [inline stylesheets/abstracts/variables.css]: here you forgot to use var

**Review by shubhamkumar648 — APPROVED** [review]
great  use  of variable name. reaminig looks clean code! change color name then good to merge

File: stylesheets/abstracts/variables.css (hunk 1 above)
Comment (tanishq20) [reply stylesheets/abstracts/variables.css]: Thanks for pointing out this.

File: stylesheets/base/typography.css [hunk 2]
```diff
@@ -1,48 +1,139 @@
...
+  color: var(--primary-color);
+}
+.txt-secondary {
+  color: var(--secondary-color);
+}
+.txt-active {
+  color: var(--active-color);
+}
+.txt-gray {
+  color: var(--grey);
```
Comment (krishanu2911) [inline stylesheets/base/typography.css:138]: just for your information here nothing wrong but you have used "gray" and in var "grey" you may get confused here while declared

**Review by krishanu2911 — COMMENTED** [review]
just added a few suggestions else everything is good to go!

File: stylesheets/base/typography.css (hunk 2 above)
Comment (tanishq20) [reply stylesheets/base/typography.css:138]: Yeah will change that.

**Review by krishanu2911 — APPROVED** [review]

**Pull request merged**